cmd/utils/flags.go

```diff
@@ -1726,6 +1731,37 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *eth.Config) {
 		cfg.NetworkId = *cfg.Genesis.GetNetworkID()
 	}
 
+	if ctx.GlobalIsSet(EthProtocolsFlag.Name) {
+		protocolVersions := SplitAndTrim(ctx.GlobalString(EthProtocolsFlag.Name))
+		if len(protocolVersions) > 0 {
+			for _, versionString := range protocolVersions {
+				version, err := strconv.ParseUint(versionString, 10, 0)
+				if err != nil {
+					continue
```

Comment:
Let's handle this error, forcing the user to provide valid values (and loudly erroring if not).

Reply:
Or ignore the error altogether, and assume that `version (==0)` won't be an existing (and thus won't be valid) as a `DefaultProtocolVersion`.

Reply:
But I think returning the error lets us provide the user with more useful context.